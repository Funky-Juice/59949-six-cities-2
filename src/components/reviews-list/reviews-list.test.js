import renderer from 'react-test-renderer';
import ReviewsList from './reviews-list';
import reviewTestObj from '../../mocks/test-review';
import offerTestObj from '../../mocks/test-offer';

const mockReviews = [reviewTestObj];
const mockOffer = offerTestObj;

it(`ReviewsList correctly renders`, () => {
  const tree = renderer
    .create(<ReviewsList
      offer={mockOffer}
      reviews={mockReviews}
      getReviews={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
