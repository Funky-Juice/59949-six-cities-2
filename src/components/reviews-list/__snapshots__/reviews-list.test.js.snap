// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsList correctly renders 1`] = `
<section
  className="property__reviews reviews"
>
  <h2
    className="reviews__title"
  >
    Reviews Â·
    <span
      className="reviews__amount"
    >
      1
    </span>
  </h2>
  <ul
    className="reviews__list"
  >
    <li
      className="reviews__item"
    >
      <div
        className="reviews__user user"
      >
        <div
          className="reviews__avatar-wrapper user__avatar-wrapper"
        >
          <img
            alt="Reviews avatar"
            className="reviews__avatar user__avatar"
            height="54"
            src="https://htmlacademy-react-2.appspot.com/six-cities/static/avatar/5.jpg"
            width="54"
          />
        </div>
        <span
          className="reviews__user-name"
        >
          Oliver
        </span>
      </div>
      <div
        className="reviews__info"
      >
        <div
          className="reviews__rating rating"
        >
          <div
            className="reviews__stars rating__stars"
          >
            <span
              style={
                Object {
                  "width": "60%",
                }
              }
            />
            <span
              className="visually-hidden"
            >
              Rating
            </span>
          </div>
        </div>
        <p
          className="reviews__text"
        >
          random text
        </p>
        <time
          className="reviews__time"
          dateTime="2019-04-24"
        >
          December
           
          2019
        </time>
      </div>
    </li>
  </ul>
</section>
`;
